src/containerbuddy/main.go

```diff
@@ -88,23 +88,26 @@ func poll(config Pollable, fn pollingFunc, command string) chan bool {
 // Implements `pollingFunc`; args are the executable we use to check the
 // application health and its arguments. If the error code on that exectable is
 // 0, we write a TTL health check to the health check store.
-func checkHealth(pollable Pollable, command string) {
+func checkHealth(pollable Pollable) {
 	service := pollable.(*ServiceConfig) // if we pass a bad type here we crash intentionally
-	if code, _ := run(command); code == 0 {
+	if code, _ := run(service.healthCheckCmd); code == 0 {
 		service.SendHeartbeat()
 	}
 }
 
 // Implements `pollingFunc`; args are the executable we run if the values in
 // the central store have changed since the last run.
-func checkForChanges(pollable Pollable, command string) {
+func checkForChanges(pollable Pollable) {
 	backend := pollable.(*BackendConfig) // if we pass a bad type here we crash intentionally
 	if backend.CheckForUpstreamChanges() {
-		run(command)
+		run(backend.onChangeCmd)
 	}
 }
 
-func getCmd(args []string) *exec.Cmd {
+func argsToCmd(args []string) *exec.Cmd {
```

Comment:
I feel like `argsToCmd` and `strToCmd` should be moved into config.go now, just because all but one of their callers are in config.go.